.buildconfig.yml

```diff
@@ -3,9 +3,13 @@ groupId: org.mozilla.telemetry
 projects:
   glean-core:
     path: glean-core/android
-    artifactId: glean-core
-    publishedArtifacts: [glean-core, glean-core-forUnitTests, glean-core-withoutLib]
-    description: 'The Glean core library'
+    artifactId: glean
+    publications:
+      - name: glean
+        type: aar
+      - name: glean-forUnitTests
+        type: jar
+    description: 'A client-side telemetry SDK for collecting metrics and sending them to the Mozilla telemetry service'
```

Comment:
nit: let's reuse the description from our docs, `The Glean SDK is a modern approach for a Telemetry library and is part of the Glean project.`. (this explicitly leaves out `client-side` and `sending to Mozilla telemetry service`